### PR TITLE
Doc Fix Pract 05 Array lesson

### PR DESCRIPTION
**Pull request by mir4oni3**
-fixed array where element should be 44
-added missing ;

### CODE REVIEW

**Review by Justsvetoslavov — APPROVED** [review]
@mir4oni3 Could you add the changes for IS/Sem.05 as well ?

Thank you!

**Comment by GeorgiTerziev02** [comment]
What is the problem with the 44 in the array? Am I missing something?

File: SI/Sem.05/README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ int arr[SIZE]; //ok!
 Достъпът става посредством индекси. Индексацията започва от 0. <br />
 
 ```c++
-int arr[] = {1, 2, 400, 4, 5}
+int arr[] = {1, 2, 400, 44, 5}
```
Comment (GeorgiTerziev02) [inline SI/Sem.05/README.md:39]: Why?

**Comment by mir4oni3** [comment]
It was 4 before I fixed it, now it is 44.

**Review by GeorgiTerziev02 — CHANGES_REQUESTED** [review]
No need to change the 44, changing the value is on the next line. The point is that initialy on index 3 we have 4, the we change it to 44

**Comment by mir4oni3** [comment]
You are right! My bad.

**Pull request merged**